### PR TITLE
Implement app installation to the aws

### PR DESCRIPTION
**Pull request by iomodo**
#### Summary
This PR implements app installation to the aws by using `/apps experimental` slash command. Command takes 3 arguments:
1. --url - an url of the released app
2. --id - AWS access key id
3. --secret - AWS secret access key
App will be installed in the provided AWS account. Installing in the Mattermost cloud will not need this data.

#### Ticket Link
  Fixes https://mattermost.atlassian.net/browse/MM-29357

### CODE REVIEW

File: server/aws/install.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+			if err != nil {
+				return errors.Wrap(err, "can't open manifest.json file")
+			}
+			defer manifestFile.Close()
+
+			data, err := ioutil.ReadAll(manifestFile)
+			if err != nil {
+				return errors.Wrap(err, "can't read manifest.json file")
+			}
+			err = json.Unmarshal(data, &mani)
```
Comment (larkox) [inline server/aws/install.go:67]: Why not use directly `json.NewDecoder(manifestFile).Decode()`?

**Review by larkox — APPROVED** [review]
LGTM. Just one small improvement.

**Review by levb — APPROVED** [review]
👍  Let's get this in and start deploying!

**Comment by levb** [comment]
@iomodo lint

**Pull request merged**